internal/pkg/plugin/sonarqube/create.go

```diff
@@ -0,0 +1,28 @@
+package sonarqube
+
+import (
+	"github.com/devstream-io/devstream/internal/pkg/plugininstaller"
+	"github.com/devstream-io/devstream/internal/pkg/plugininstaller/helm"
+	"github.com/devstream-io/devstream/pkg/util/log"
+)
+
+func Create(options map[string]interface{}) (map[string]interface{}, error) {
+	// Initialize Operator with Operations
+	operator := &plugininstaller.Operator{
+		PreExecuteOperations: plugininstaller.PreExecuteOperations{
+			helm.SetDefaultConfig(&defaultHelmConfig),
+			helm.Validate,
+		},
+		ExecuteOperations:   helm.DefaultCreateOperations,
+		TerminateOperations: helm.DefaultTerminateOperations,
```

Comment:
Since `Update` calls `Create`, if it failed in `update`, tools will be deleted.

<img width="668" alt="image" src="https://user-images.githubusercontent.com/36830265/190546388-7a186721-e4a4-4d96-b034-a3b67358f5e1.png">